src/VisualStudio/InteractiveServices/Interactive/VsInteractiveWindowPackage.cs

```diff
@@ -52,6 +52,8 @@ protected override void Initialize()
 
             _componentModel = (IComponentModel)GetService(typeof(SComponentModel));
             _interactiveWindowProvider = _componentModel.DefaultExportProvider.GetExportedValue<TVsInteractiveWindowProvider>();
+            KnownUIContexts.ShellInitializedContext.WhenActivated(() =>
+                _componentModel.GetService<HACK_ThemeColorFixer>());
```

Comment:
@ManishJayaswal based on the code of `HACK_ThemeColorFixer `, it affects both tooltip and immediate , although I don't know what an immediate is...